recipe/meta.yaml

```diff
@@ -14,24 +14,28 @@ source:
 build:
   number: 0
   script:
-    - python setup.py build_ext --compiler=mingw32  # [win and py2k]
+    - >-                                                # [win and py27]
+          python setup.py build_ext --compiler=mingw32  # [win and py27]
+          -I%LIBRARY_PREFIX%\\mingw-w64\\include        # [win and py27]
```

Comment:
Seems to not find the `lzma.h` header still even with this. 😕

ref: https://ci.appveyor.com/project/conda-forge/backports-lzma-feedstock/build/1.0.49/job/h5rf2n2opqlg9qgo